internal/gcs-sidecar/handlers.go

```diff
@@ -385,33 +381,28 @@ func (b *Bridge) modifySettings(req *request) (err error) {
 			}
 			return nil
 
-		case guestresource.ResourceTypeCWCOWCombinedLayers:
```

Comment:
@ambarve As discussed, this was added intentionally. It is just logging in this change, but changes need containerID and is being used for enforcement. Please revert this commit.